<ui version="4.0" >
 <class>CreateCase</class>
 <widget class="QDialog" name="CreateCase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Create Case</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="13" column="0" colspan="2" >
    <widget class="QGroupBox" name="GroupButtons" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QPushButton" name="buttonOk" >
        <property name="text" >
         <string>&amp;Ok</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="buttonApply" >
        <property name="text" >
         <string>&amp;Apply</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="buttonCancel" >
        <property name="text" >
         <string>&amp;Cancel</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="buttonHelp" >
        <property name="text" >
         <string>&amp;Help</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0" colspan="2" >
    <widget class="QGroupBox" name="GBBoundaryA" >
     <property name="minimumSize" >
      <size>
       <width>548</width>
       <height>300</height>
      </size>
     </property>
     <property name="title" >
      <string>Analytic Boundary</string>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>410</x>
        <y>20</y>
        <width>120</width>
        <height>110</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="PBBoundaryAnNew" >
         <property name="text" >
          <string>New</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBBoundaryAnHelp" >
         <property name="text" >
          <string>Help</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="TWBoundary" >
      <property name="geometry" >
       <rect>
        <x>4</x>
        <y>20</y>
        <width>371</width>
        <height>270</height>
       </rect>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="showGrid" >
       <bool>true</bool>
      </property>
      <property name="rowCount" >
       <number>0</number>
      </property>
      <property name="columnCount" >
       <number>1</number>
      </property>
      <column>
       <property name="text" >
        <string>a_virer</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="Name" >
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LECaseName" >
       <property name="minimumSize" >
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="Directory" >
       <property name="text" >
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushDir" >
       <property name="text" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEDirName" >
       <property name="minimumSize" >
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="Mesh_2" >
       <property name="text" >
        <string>Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushFichier" >
       <property name="text" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEFileName" >
       <property name="minimumSize" >
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QGroupBox" name="GBTypeConf" >
     <property name="title" >
      <string>Conformity Type</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RBConforme" >
        <property name="text" >
         <string>Conformal</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBNonConforme" >
        <property name="text" >
         <string>Non conformal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QGroupBox" name="GBTypeNoConf" >
     <property name="title" >
      <string>Non Conformal option</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RB1NpM" >
        <property name="text" >
         <string>1 hanging node per mesh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RB1NpA" >
        <property name="text" >
         <string>1 node per edge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBQuelconque" >
        <property name="text" >
         <string>free</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="CBBoundaryD" >
       <property name="text" >
        <string>Discrete Boundary </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CBBoundaryA" >
       <property name="text" >
        <string>Analytic Boundary </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" >
    <widget class="QGroupBox" name="GBBoundaryD" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Discrete Boundary</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="4" >
       <widget class="QPushButton" name="PBBoundaryDiHelp" >
        <property name="text" >
         <string>Help</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="PBBoundaryDiNew" >
        <property name="text" >
         <string>New</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="PBBoundaryDiEdit" >
        <property name="text" >
         <string>Edit</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QComboBox" name="CBBoundaryDi" >
        <property name="currentIndex" >
         <number>-1</number>
        </property>
        <property name="sizeAdjustPolicy" >
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
